### DISCUSSION TITLE
Anyone figure out how to default a homepod -> AirPlay?

### DISCUSSION THREAD

**Discussion by osheroff** [mikebrady/shairport-sync#1673] [Q&A]
When I ask siri (via a homepod mini) to play music via a voice command, it always defaults to the the homepod.  I can go in an choose the shairport device and playback switches to there. 

I can't seem to get "play music on [name of the shairplort]" or "add [name of shairport]" working either, the homepod doesn't seem to be able to find the speaker.

Does anyone have good experiences getting this to work?

**Comment by mikebrady** [comment]
Thanks for the post. The comment [here](https://github.com/mikebrady/shairport-sync/issues/1514#issuecomment-1221253518) might help.
